src/client/components/ContactForm/ContactForm.styles.css

```diff
@@ -1,38 +1,46 @@
-form {
+.formContact {
   display: grid;
   padding: 1em;
-  background: #f9f9f9;
+  background-color: #f0eeeef3;
   border: 1px solid #c1c1c1;
```

Comment:
Jai, maybe for easier styling in other page, background color change or be a prop. about border maybe remove it for better styling in other components.

Reply:
Done